deployments/UnityMixpanel/Assets/Mixpanel/Mixpanel.cs

```diff
@@ -668,4 +674,34 @@ void Update()
         #endregion
     }
 
+    class IntegrationReporter : MonoBehaviour {
+        public string token = null;
+
+        void Update() {
+            if (token == null || token.Length == 0) return;
+
+            enabled = false;
+            string url = BuildRequestURL();
+            var request = new WWW(url);
+            StartCoroutine(WaitForRequest(request));
+        }
+
+        string BuildRequestURL() {
+            string body = "{\"event\":\"Integration\",\"properties\":{\"token\":\"85053bf24bba75239b16a601d9387e17\",\"mp_lib\":\"unity\",\"distinct_id\":\"" + token +"\"}}";
+            byte[] bytes = Encoding.UTF8.GetBytes(body);
+            string encoded = Convert.ToBase64String(bytes);
```

Comment:
Maybe let's use: "e1wiZXZlbnRcIjpcIkludGVncmF0aW9uXCIsXCJwcm9wZXJ0aWVzXCI6e1widG9rZW5cIjpcIjg1MDUzYmYyNGJiYTc1MjM5YjE2YTYwMWQ5Mzg3ZTE3XCIsXCJtcF9saWJcIjpcInVuaXR5XCIsXCJkaXN0aW5jdF9pZFwiOlwiIiArIHRva2VuICsiXCJ9fQ==" instead of encoding and revealing